### PR TITLE
Update POST for view

### PR DESCRIPTION
**Pull request by bessbd**
## Overview
In https://github.com/apache/couchdb/pull/2338 the way POST view works is altered so that the same functionality is available as in GET view.

This commit updates the POST view documentation to reflect this new behavior.

This depends on https://github.com/apache/couchdb/pull/2338 .

## Testing recommendations

Execute the tests seen in https://github.com/apache/couchdb/pull/2338 .

## Related Pull Requests
* https://github.com/apache/couchdb/pull/2338 

## Checklist

- [x] Documentation is written and is accurate;
- [x] `make check` passes with no errors
- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/master/rebar.config.script) with the commit hash once this PR is rebased and merged

### CODE REVIEW

File: src/api/ddoc/views.rst [hunk 1]
```diff
@@ -157,13 +157,10 @@
...
 .. http:post:: /{db}/_design/{ddoc}/_view/{view}
-    :synopsis: Returns certain rows for the specified stored view
+    :synopsis: Returns results for the specified stored view
 
     Executes the specified view function from the specified design document.
-    Unlike :get:`/{db}/_design/{ddoc}/_view/{view}` for accessing views, the
-    :method:`POST` method supports the specification
-    of explicit keys to be retrieved from the view results. The remainder of
-    the :method:`POST` view functionality is identical to the
+    :method:`POST` view functionality is identical to the
```
Comment (davisp) [inline src/api/ddoc/views.rst]: This is a bit misleading. I'd reword this to be something along the lines of:

```
:method:`POST` view functionality supports identical parameters and behavior
as specified in :get:`/{db}/_design/{ddoc}/_view/{view}` API but allows for the
query string parameters to be supplied as keys in JSON object in the body
of the `POST` request.
```

Comment (bessbd) [reply src/api/ddoc/views.rst]: Good call! I'll push a commit for this soon.

Comment (bessbd) [reply src/api/ddoc/views.rst]: Fixed in 3cf6106a6a320c1b3f35ba29cd96c1def42db84d

File: src/api/ddoc/views.rst [hunk 2]
```diff
@@ -157,14 +157,13 @@
...
     Executes the specified view function from the specified design document.
-    Unlike :get:`/{db}/_design/{ddoc}/_view/{view}` for accessing views, the
-    :method:`POST` method supports the specification
-    of explicit keys to be retrieved from the view results. The remainder of
-    the :method:`POST` view functionality is identical to the
-    :get:`/{db}/_design/{ddoc}/_view/{view}` API.
+    :method:`POST` view functionality supports identical parameters and behavior
+    as specified in :get:`/{db}/_design/{ddoc}/_view/{view}` API but allows for the
+    query string parameters to be supplied as keys in JSON object in the body
+    of the `POST` request.
```
Comment (davisp) [inline src/api/ddoc/views.rst]: Grammar nit:

```
"as specified in the :get:`...`"
```

Currently missing "the" in the text.

```
supplied as keys in a JSON
```

Currently missing "a"

Comment (bessbd) [reply src/api/ddoc/views.rst]: Fixed in c0d38dab4fc926554d8741bebfbd9bd08fff72d9

File: src/api/ddoc/views.rst [hunk 3]
```diff
@@ -157,14 +157,13 @@
     transfer size for attachments.
 
 .. http:post:: /{db}/_design/{ddoc}/_view/{view}
-    :synopsis: Returns certain rows for the specified stored view
+    :synopsis: Returns results for the specified stored view
```
Comment (flimzy) [inline src/api/ddoc/views.rst]: Very minor:  Are there unstored views?  I think this could be shortened to just

```suggestion
    :synopsis: Returns results for the specified view
```

**Comment by flimzy** [comment]
+1

File: src/api/ddoc/views.rst (hunk 3 above)
Comment (bessbd) [reply src/api/ddoc/views.rst]: Good call! I'll commit your suggestion.

**Comment by bessbd** [comment]
Thank you for the review, @davisp and @flimzy !

**Comment by bessbd** [comment]
@davisp : I think I've addressed all review comments on this change. Can you please do a next / final pass?

**Comment by bessbd** [comment]
Thank you for the approval, @davisp !

**Comment by bessbd** [comment]
Thank you for the merge, @garrensmith !
